modules/statics/src/account.ts

```diff
@@ -409,6 +411,49 @@ export function terc20(
   return erc20(name, fullName, decimalPlaces, contractAddress, asset, features, prefix, suffix, network);
 }
 
+export function nft(
```

Comment:
Would it make sense to follow the naming convention of the other functions that are similar to somthing like `erc721()` and `terc721()`? Or is this more agnostic?